xyz-hub-service/src/main/java/com/here/xyz/hub/connectors/RpcClient.java

```diff
@@ -394,10 +394,10 @@ private void validateResponsePayload(Marker marker, final Typed payload) throws
           throw new HttpException(TOO_MANY_REQUESTS,
               "The connector cannot process the message due to a limitation in an upstream service or a database.", errorResponse.getErrorDetails());
         case ILLEGAL_ARGUMENT:
+        case EXCEPTION:
```

Comment:
Transforming all EXCEPTION-responses into BAD_REQUESTs could be ending up in forwarding too many internal errors as 400s. In general, the connector responds with EXCEPTION also for internal errors which should end up as 5XX. It could be that instead we need another error type for connector responses.